frontend/js/src/common/listens/AddToPlaylist.tsx

```diff
@@ -100,7 +101,7 @@ export default NiceModal.create((props: AddToPlaylistProps) => {
               message={
                 <>
                   Successfully added <i>{trackToSend.title}</i> to playlist{" "}
-                  <a href={playlistIdentifier}>{playlistName}</a>
+                  <Link to={playlistIdentifier}>{playlistName}</Link>
```

Comment:
Sorry, I was wrong!
My suggestion would work fine in production, but not if you are on the beta or test websites or developing locally on your machine.
Instead, this is what we want:
```suggestion
                  <Link to={`/playlist/${playlistId}`}>{playlistName}</Link>
```